openlibrary/accounts/model.py

```diff
@@ -681,13 +736,15 @@ def audit_accounts(email, password, test=False):
             ol_account.link(audit['link'])
             stats.increment('ol.account.xauth.auto-linked')
```

Comment:
Seems like this whole branch starting with `# Links the accounts if they can be and are not already` is not needed for the production case. Haven't we already hit every case up above?